apps/web/src/constants/routes.enum.ts

```diff
@@ -0,0 +1,24 @@
+export enum ROUTES {
+  HOME = '/',
+  AUTH_SIGNUP = '/auth/signup',
+  AUTH_LOGIN = '/auth/login',
+  AUTH_RESET_REQUEST = '/auth/reset/request',
+  AUTH_RESET__TOKEN = '/auth/reset/:token',
```

Comment:
Can we refactor this to only have one `_` seems like there is a double underscore here:)

Reply:
Hi added that extra underscore intentionally for differentiating URL param from rest of the URL.

Reply:
If you prefer having only one underscore then we can make changes accordingly.

Reply:
Two underscores together are definitely a problem regarding readability (think in people developing with some visual impairment, would be harder for them to notice and potentially frustrating).

Reply:
okay. I'll change two underscores with a single underscore.